tools/fix_timepoint_date_problems.php

```diff
@@ -454,11 +454,11 @@ function diagnose($sessionID, $dateType=null, $newDate=null)
 
     // check if the timepoint is started before attempting to make changes to it
     if ($timePoint->getCurrentStage() == 'Not Started' || empty($stageList['screening']['status'])) {
-        throw new LorisException("Error: Cannot diagnose the non-started timepoints!");
+        throw LorisException("Error: Cannot diagnose the non-started timepoints!");

```

Comment:
PHP Fatal error:  Uncaught Error: Call to undefined function LorisException() in /var/www/loris/tools/fix_timepoint_date_problems.php:457